src/usr/local/bin/prepare-tool

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -e
+
+. /usr/local/buildpack/util.sh
+
+require_distro
+require_root
+require_tool "$@"
```

Comment:
This probably fails when no version is found

https://github.com/containerbase/buildpack/blob/48c5719e1537bc1c2abc4466a31fb46a831dcade/src/usr/local/buildpack/util.sh#L214-L217

So we maybe need to refactor / split that function too.

Reply:
Strange, it didn't failed as expected 😕

Reply:
It did not, but I will investigate, because I had the same thought